UndertaleModTool/Editors/UndertaleEmbeddedTextureEditor.xaml.cs

```diff
@@ -154,29 +174,41 @@ private void Import_Click(object sender, RoutedEventArgs e)
             {
                 try
                 {
-                    Bitmap bmp;
-                    using (var ms = new MemoryStream(TextureWorker.ReadTextureBlob(dlg.FileName)))
+                    GMImage image;
+                    if (System.IO.Path.GetExtension(dlg.FileName).Equals(".png", StringComparison.InvariantCultureIgnoreCase))
```

Comment:
Why it needs to be this complicated, can't you just:
```suggestion
                    if (System.IO.Path.GetExtension(dlg.FileName) == ".png")
```

Reply:
It needs to be case-insensitive (e.g., if the filename ends in `.PNG` instead of `.png`).

Reply:
Is there a difference between them?